.github/workflows/publish.yml

```diff
@@ -0,0 +1,96 @@
+name: Build and publish 📦
```

Comment:
When is this workflow triggered?

Reply:
When we add a release on Github?

Reply:
yes, exactly. only when we publish a new release with a tagged commit.